ballerina-test-automation/gradle.properties

```diff
@@ -1,7 +1,7 @@
 swan-lake-latest-version=swan-lake-2201.11.0
 swan-lake-latest-spec-version=2024R1
 swan-lake-latest-version-display-text=2201.11.0
-swan-lake-latest-tool-version=1.4.3
-latest-tool-version=1.4.3
+swan-lake-latest-tool-version=1.4.5
+latest-tool-version=1.4.5
```

Comment:
Why do we need to properties?

Reply:
For a gradle project, it is the best practice to maintain property values like this instead of passing environment variables. We usually pass environment variables to change something in the build.